cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -23,3 +31,44 @@ spec:
         resources:
           requests:
             storage: 10Gi
+  cephVersion:
+    image: ceph/ceph:v14.2.1-20190430
+    allowUnsupported: false
+  dashboard:
+    enabled: true
+  network:
+    hostNetwork: false
+  storage:
+    storageClassDeviceSets:
+    - name: cluster1-set1
+      count: 3
+      resources:
+        requests:
+          cpu: 2
+          memory: 4Gi
+      # placement:
+      #   podAntiAffinity:
+      #     preferredDuringSchedulingIgnoredDuringExecution:
+      #     - weight: 100
+      #       podAffinityTerm:
+      #         labelSelector:
+      #           matchExpressions:
+      #           - key: "rook.io/cluster"
+      #             operator: In
+      #             values:
+      #               - cluster1
+      #           topologyKey: "failure-domain.beta.kubernetes.io/zone"
+      volumeClaimTemplates:
+      - metadata:
+          name: template-1
+        spec:
+          resources:
+            requests:
+              storage: 10Gi
+          # IMPORTANT: Change the storage class depending on your environment
+          storageClassName: local-storage
+          volumeMode: Block
+          accessModes:
+            - ReadWriteOnce
+    useAllNodes: true
```

Comment:
let's also remove useAllNodes and useAllDevices from this example

Reply:
Removed.